x/concentrated-liquidity/swaps.go

```diff
@@ -374,7 +374,8 @@ func (k Keeper) computeOutAmtGivenIn(
 	}
 
 	// Add spread reward growth per share to the pool-global spread reward accumulator.
-	spreadRewardAccumulator.AddToAccumulator(sdk.NewDecCoins(sdk.NewDecCoinFromDec(tokenInMin.Denom, swapState.spreadRewardGrowthGlobal)))
+	spreadRewardGrowth := sdk.NewDecCoinFromDec(tokenInMin.Denom, swapState.globalSpreadRewardGrowthPerUnitLiquidity)
+	spreadRewardAccumulator.AddToAccumulator(sdk.NewDecCoins(spreadRewardGrowth))
```

Comment:
```suggestion
	spreadRewardGrowthPerUnitOfLiquidityCoin := sdk.NewDecCoinFromDec(tokenInMin.Denom, swapState.globalSpreadRewardGrowthPerUnitLiquidity)
	spreadRewardAccumulator.AddToAccumulator(sdk.NewDecCoins(spreadRewardGrowthPerUnitOfLiquidityCoin))
```

Reply:
Another option is to collapse this addition into a single line to be consistent with swap in given out